tests/unit/services/LiqualityService.spec.ts

```diff
@@ -73,7 +73,7 @@ describe('Liquality Service:', () => {
         expect(mockedBitcoinProvider.enable.notCalled).to.be.true;
         // eslint-disable-next-line no-unused-expressions
         expect(request.calledTwice).to.be.true;
-      });
+      }).catch((e) => expect(e.message).to.be.eql('Liquality software wallet not installed on your browser'));
```

Comment:
please add a test case where `enable` method from `mockedBitcoinProvider` reject insted of resolve, this line could be tested in this case

Reply:
Unit tests adjusted. Added a new case when exception is raised.